packages/data-stores/src/reader/constants.ts

```diff
@@ -21,3 +21,9 @@ export enum SiteSubscriptionsSortBy {
 	LastUpdated = 'last_updated',
 	DateSubscribed = 'date_subscribed',
 }
+
+export enum SubscribersFilterBy {
```

Comment:
Should this be in the Reader package? My impression was that we store everything for subscription management here, since it's shared between the standalone portal and the reader integration. I think for Subscriber Page it can live closer to the subscriber page code.